packages/web/tailwind.config.js

```diff
@@ -2,7 +2,7 @@ const colors = require('tailwindcss/colors')
 
 module.exports = {
   purge: [
-    'src/views/**/*.pug'
+    // 'src/views/**/*.pug'
```

Comment:
Don't forgot to enable back! Trello card to use JIT instead has been created. Should speed up things significantly. Also nanocss should probably only be used if NODE_ENV=production